velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -573,4 +573,21 @@ TEST_F(DecodedVectorTest, wrapOnConstantEncoding) {
   }
 }
 
+TEST_F(DecodedVectorTest, noValues) {
```

Comment:
nit: perhaps, add a comment to clarify that this is testing flat vector of all nulls with nullptr values buffer.